src/services/item/ItemWrapper.ts

```diff
@@ -67,11 +67,18 @@ export class ItemWrapper {
     for (const i of items) {
       const { permission = null } = memberships.data[i.id] ?? {};
       const thumbnails = itemsThumbnails?.[i.id];
+
+      // sort tags to retrieve most relevant (highest) tag first
+      const thisTags = tags?.data?.[i.id];
```

Comment:
I'm not a big fan of `thisTags` ^^, what do you think about my suggestion?
```suggestion
      const itemTags = tags?.data?.[i.id];
```